Backend/app/services/alert_monitoring_service.py

```
@@ -0,0 +1,220 @@
+        self.is_running = False
+        logger.info("Stopping alert monitoring service")
+
+    async def _check_all_alerts(self):
+        """Check all active alerts and trigger notifications if needed."""
+        db = None
+        try:
+            db = next(get_db())
+            
+            # Get all active alerts
+            active_alerts = db.query(AlertConfig).filter(
+                AlertConfig.is_active == True
```

Ruff (0.13.1)

Avoid equality comparisons to `True`; use `AlertConfig.is_active:` for truth checks

Replace with `AlertConfig.is_active`

(E712)